### PR TITLE
fix(auth): Auth pages responsiveness

### PR DESCRIPTION
**Pull request by jorgelrj**
Fixed the bug you were commenting on the video 😉
Also, from a quick look on the project, you should really look into implementing the inputs decoration inside the input widget instead of using multiple containers (https://dart-lang.github.io/linter/lints/sized_box_for_whitespace.html)

### CODE REVIEW

File: app/exerlog/lib/UI/login_screen/login_page.dart [hunk 1]
```diff
@@ -45,140 +41,127 @@ class _LoginPageState extends State<LoginPage> {
...
+                    onPressed: () async {
+                      if (index == 0) {
+                        // login with email and password
+                        if (loginData.password != '' && loginData.email != '') {
+                          final user = await EmailSignup.signInWithEmailAndPassword(
+                            loginData.email,
+                            loginData.password,
+                          );
+
+                          if (user != null) {
```
Comment (jarg1023) [inline app/exerlog/lib/UI/login_screen/login_page.dart:119]: I personally would do a guard clause, easier to read:

```dart
if (user == null) {
    show error message to user
    return
}

whatever needs to happen after signIn...
```

Comment (jorgelrj) [reply app/exerlog/lib/UI/login_screen/login_page.dart:119]: I limited myself to make only UI changes there, all the logic is still the same as it was before
But I agree with you, there's a lot of room for improvement there

File: app/exerlog/lib/UI/login_screen/login_page.dart [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
-import '../maxes/max_builder.dart';
 
 class LoginPage extends StatefulWidget {
-  LoginPage(this.title, {Key? key}) : super(key: key);
-  String title = "0";
+  final String title;
+
+  const LoginPage({
+    Key? key,
+    this.title = "0",
```
Comment (Scount) [inline app/exerlog/lib/UI/login_screen/login_page.dart:16]: With this change an error occurs in the file 'main.dart' one line 60. 
The value `title` is a named parameter. Change in 'main.dart' from `LoginPage('1')` to `LoginPage(title:'1')` or `LoginPage()`

Comment (jorgelrj) [reply app/exerlog/lib/UI/login_screen/login_page.dart:16]: Forgot to add the file to the commit, thanks

**Comment by KalleHallden** [comment]
Man this was awesome! Thank you and thank you to @Scount and @jorgelrj as well

**Pull request merged**